tests/unit/connect.c

```diff
@@ -70,11 +70,11 @@ int main(int argc, char *argv[])
   // Now that we know everything is good... lets push it.
   drizzle_close(con);
 
-  int limit= 20;
   while (--limit)
   {
-    ret= drizzle_connect(con);
     ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "%s(%s)", drizzle_error(con), drizzle_strerror(ret));
+  int limit = 20;
```

Comment:
It seems like some of the changes are shown out of order/wrongly. It compiles and the location of `limit` is where it should be. 
I will try to see if it changes after force pushing and rebasing on latest v5.4.x